package.json

```diff
@@ -5,8 +5,9 @@
     "install:all": "npm install && cd server && npm install && cd ../client && npm install",
     "start": "concurrently \"npm run start:server\" \"npm run start:client\"",
     "start:docker": "cross-env NODE_ENV=docker npm start",
-    "start:server": "cd server && nodemon server",
+    "start:server": "cd server && npm run start",
     "start:client": "npm start --prefix client",
+    "build:server": "cd server && npm run build",
```

Comment:
please explain me this command

Reply:
This command transform all our .ts files to .ts files and put them in ./build directory on the server. Then we can run node build/server.js in our production version. If we do not plan to run our client build and server build in same time I'll remove this command